### ISSUE TITLE
Error resolving CustomToolTip - 'customtooltip' depends on an unstable-version package 'swizzlehelper'

### ISSUE THREAD

**Issue by jeffaburt** [chipjarred/CustomToolTip#3]
Receiving the following error when trying to resolve CustomToolTip via SPM:

```
Dependencies could not be resolved because package 'customtooltip' is required using a stable-version but 'customtooltip' depends on an unstable-version package 'swizzlehelper' and root depends on 'CustomToolTip' 1.0.0..<2.0.0.
'CustomToolTip' {1.0.0..<1.0.2, 1.0.3..<2.0.0} cannot be used because package 'customtooltip' is required using a stable-version but 'customtooltip' depends on an unstable-version package 'swizzlehelper' and package 'customtooltip' is required using a stable-version but 'customtooltip' depends on an unstable-version package 'swizzlehelper'.
'CustomToolTip' {1.0.0, 1.0.3..<2.0.0} cannot be used because no versions of 'CustomToolTip' match the requirement 1.0.3..<2.0.0 and package 'customtooltip' is required using a stable-version but 'customtooltip' depends on an unstable-version package 'swizzlehelper'.
```

I believe this is because we're referencing `.branch("main")` in https://github.com/chipjarred/CustomToolTip/blob/main/Package.swift#L18. Is there a reason not to change this to the https://github.com/chipjarred/SwizzleHelper/releases/tag/1.0.0 tag?

In the meantime, I'll just add CustomToolTip using the `main` branch instead of using the latest tag. Thanks in advance for the help!

**Comment by mrommel** [comment]
Please fix this using the above comment or update the description

**Comment by chipjarred** [comment]
Sorry I haven't been available to act on this more promptly.  @jeffaburt, you are correct that it's because of the named branch rather version tag.  I think I used the named branch because it was easier to keep changes in sync while making local changes and testing, but I must have forgotten to change it back when committing and pushing.  I'll fix it shortly.  Thank you for reporting it!

**Comment by chipjarred** [comment]
Fixed now with version 1.0.3 which now explicitly depends on 1.0.1 of SwizzleHelper.
